gulpfile.js

```diff
@@ -58,22 +57,22 @@ const paths = {
 // Try to ensure that all processes are killed on exit
 const spawned = [];
 process.on('exit', () => {
-  spawned.forEach((pcs) => {
+  spawned.forEach(pcs => {
```

Comment:
I do honestly prefer even 1-ary anonymous arrow functions to have parens, but it's a mild preference.